apps/meteor/client/views/omnichannel/directory/chats/ChatTable.tsx

```diff
@@ -167,12 +161,26 @@ const ChatTable: FC<{ setChatReload: Dispatch<SetStateAction<any>> }> = ({ setCh
 		[onRowClick],
 	);
 
+	const result = useCurrentChats(query);
+
+	useEffect(() => {
+		setChatReload?.(() => result.refetch);
```

Comment:
Instead of using the refetch we should invalidate the query on reload https://tanstack.com/query/v4/docs/react/guides/query-invalidation